### PR TITLE
Add minutes_left_on_reserve to status.v1 response

### PR DESCRIPTION
**Pull request by mrstennett**
Needed for change in the expedia-api-adapter to for https://github.com/ingresso-group/expedia-api-adapter/issues/103

### CODE REVIEW

File: status.go [hunk 1]
```diff
@@ -4,10 +4,11 @@ import "time"
...
 // StatusResult describes the current state of a transaction.
 type StatusResult struct {
-	Languages        []string            `json:"language_list"`
-	Trolley          Trolley             `json:"trolley_contents"`
-	ReserveDatetime  time.Time           `json:"reserve_iso8601_date_and_time"`
-	PurchaseDatetime time.Time           `json:"purchase_iso8601_date_and_time"`
-	CurrencyDetails  map[string]Currency `json:"currency_details"`
-	Status           string              `json:"transaction_status"`
+	Languages            []string            `json:"language_list"`
+	MinutesLeftOnReserve float64             `json:"minutes_left_on_reserve"`
```
Comment (mrstennett) [inline status.go:8]: Only this has changed

**Comment by mrstennett** [comment]
Not sure what's going on with the test though

**Comment by cakemanny** [comment]
> Not sure what's going on with the test though

are you subscribed in the circleci UI?

that was a problem we were having following Aaron and/or Konrad leaving with their projects

**Review by cakemanny — APPROVED** [review]
can't argue with this

**Pull request merged**